packages/kv-asset-handler/package.json

```diff
@@ -20,36 +20,36 @@
 	},
 	"license": "MIT OR Apache-2.0",
 	"author": "wrangler@cloudflare.com",
+	"type": "module",
 	"main": "dist/index.js",
 	"types": "dist/index.d.ts",
 	"files": [
 		"src",
-		"dist",
-		"!src/test",
-		"!dist/test"
+		"dist"
 	],
 	"scripts": {
-		"build": "tsc -d",
+		"build": "tsup",
 		"check:lint": "eslint . --max-warnings=0",
 		"check:type": "tsc",
 		"prepack": "npm run build",
 		"pretest": "npm run build",
-		"test": "ava dist/test/*.js --verbose",
-		"test:ci": "npm run build && ava dist/test/*.js --verbose"
+		"test": "vitest",
+		"test:ci": "vitest run --inspect=9229 --no-file-parallelism"
 	},
 	"dependencies": {
 		"mime": "^3.0.0"
 	},
 	"devDependencies": {
-		"@ava/typescript": "^4.1.0",
 		"@cloudflare/eslint-config-shared": "workspace:*",
+		"@cloudflare/vitest-pool-workers": "^0.7.0",
 		"@cloudflare/workers-types": "catalog:default",
 		"@types/mime": "^3.0.4",
 		"@types/node": "catalog:default",
 		"@types/service-worker-mock": "^2.0.1",
-		"ava": "^6.0.1",
 		"eslint": "catalog:default",
-		"service-worker-mock": "^2.0.5"
+		"service-worker-mock": "^2.0.5",
```

Comment:
Do we still need service-worker-mock if it's now running in vitest-pool-workers?

Reply:
Probably not. Let me try removing it.